### DISCUSSION TITLE
How Can I enable the thumbnails of file *.epub in windows explorer - OS: windows 11

### DISCUSSION THREAD

**Discussion by PaoloSi** [sumatrapdfreader/sumatrapdf#3357] [Q&A]
How Can I enable the thumbnails of file *.epub in windows explorer - OS: windows 11

**Comment by GitHubRulesOK** [comment]
you could compile with that ability switched on (it is not ON in any release or pre-release binaries) for more on the topic see https://github.com/sumatrapdfreader/sumatrapdf/issues/840
or use another previewer like https://github.com/fire-eggs/DarkThumbs but do check it too may have issues https://github.com/fire-eggs/DarkThumbs/issues
MS seem to have removed one they too seem to have classed as experimental
